### DISCUSSION TITLE
优化"动态反折叠"的功能说明

### DISCUSSION THREAD

**Discussion by XMB-7** [the1812/Bilibili-Evolved#2743] [功能建议 / Ideas]
目前是"自动展开被折叠的动态", 此说明较难清晰解释这是啥功能, 我还是在讨论区里一个题文不相关的帖子里才偶然发现了所指, 因此建议改为诸如"某作者在短时间内大量更新作品会被自动折叠, 勾选此项将会自动取消此类动态折叠"之类.

**Comment by Geeyun-JY3** [comment]
其实啊，被折叠的动态还可能是受到破站审核制裁的动态，就是这类动态的反折叠该功能还没适配，见 https://github.com/the1812/Bilibili-Evolved/issues/2529#issuecomment-992128167
